### PR TITLE
Prevent crash on `sigil_prefix` and `sigil_suffix` (OTP 27)

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
# Motivation

`rebar3_hank` crashes when trying to analyse a file that contains

```erlang
~"""
This is a string
"""
```

# Missing tests

~I can't seem to be able to create a test for this. It's possible there's some type of pre-processing on the input. Please advise.~

# Further considerations

We're using `rebar3_hank` in this lib., but that's also crashing due to this lib. We could create a task to re-incorporate it later, if you think it worth it. The same can be done for the formatter, since that's also crashing, even with the `-format ignore` that I just removed.

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
I was able to conjure up a test, though this is a very strange occurrence (it's late so I guess I must be sleepy). Without the `?assertMatch` there's no issue (I got this from our failing test suite). With it there's an issue. I could just change the test, but it seems more appropriate to update `katana_code`.

File: rebar.config [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {profiles,
  [{test, [{cover_enabled, true}, {cover_opts, [verbose]}, {ct_opts, [{verbose, true}]}]}]}.
 
-{alias, [{test, [compile, format, hank, lint, xref, dialyzer, ct, cover, ex_doc]}]}.
+{alias, [{test, [compile, lint, xref, dialyzer, ct, cover, ex_doc]}]}.
```
Comment (elbrujohalcon) [inline rebar.config:11]: We should definitely write up a ticket to recover these 2 tools once they incorporate the new katana version we'll create with these changes.

**Review by elbrujohalcon — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Any chance this gets released soon so we can update tools depending on `katana_code`? Thanks.

**Comment by elbrujohalcon** [comment]
Done 👍🏻 : https://github.com/inaka/katana-code/releases/tag/2.1.1

**Pull request merged**